plugins/BEdita/Core/src/Model/Action/ListRelatedObjectsAction.php

```diff
@@ -44,6 +44,9 @@ protected function initialize(array $config)
                 $objectType = current($objectTypes);
             }
             $this->ListAction = new ListObjectsAction(compact('table', 'objectType'));
+        } elseif ($this->Association->getName() === 'Parents') {
```

Comment:
This is totally fine. I'm just wondering if we can perform some kind of a more flexible check, like `$this->Association->getTarget() instanceof ObjectsTable` or something like that… @batopa ?

(Same would apply to `ListRelatedFoldersAction::initialize()`)

Reply:
A more flexible approach should be better for sure but  for `Parents` association `$this->Association->getTarget()` would be `FoldersTable` while for `Children` association would be `ObjectsTable`. 

Maybe we could check that `$this->Association->getTarget() instanceof ObjectsTable || $this->Association->getTarget() instanceof \BEdita\Core\ORM\Inheritance\Table `. 
In this way we could keep this check only in `ListRelatedObjectsAction` and remove the check from `ListRelatedFoldersAction`.